src/index.js

```diff
@@ -79,7 +80,7 @@ function ParseServer({
   cloud,
   collectionPrefix = '',
   clientKey = '',
-  javascriptKey = '',
+  javascriptKey = randomString(20),
```

Comment:
yeah that's not ideal, but the cloud code SDK breaks when no JS key is set, we could throw if JS key is not set and .cloud is set